bob-apps/bin/bobc.rs

```diff
@@ -113,3 +114,13 @@ fn get_key_value(matches: &'_ ArgMatches<'_>) -> u64 {
         .parse()
         .expect("key must be u64")
 }
+
+fn get_key(k: u64) -> Vec<u8> {
+    // TODO move to static
+    let key_size = option_env!("BOB_KEY_SIZE")
+        .map_or(Ok(8), str::parse)
+        .expect("Could not parse BOB_KEY_SIZE");
+    let mut data = k.to_le_bytes().to_vec();
+    data.extend(repeat(0).take(key_size - data.len()));
```

Comment:
It is faster and safer to do this inside `if` only when `key_size - data.len() > 0` (`key_size` in practice can be less than `data.len()`)

Reply:
Fixed